site/en/docs/extensions/reference/storage/index.md

```diff
@@ -2,26 +2,66 @@
+- Content scripts can directly access chrome.storage.
+- Stores JSON serializable values as object properties.
+- Storage is asynchronous with bulk read and write operations.
+- Storage data persists when the user clears the cache and browsing history.
+- Available in the service worker, content scripts, and other extension contexts.
+- A user's extension settings persist even when using [split incognito][incognito].
+- Offers an exclusive read-only [managed storage area][manifest-storage] for enterprise policies.
 
-## Manifest
+{% Details %}
+{% DetailsSummary %}
+💡 Can extensions use Window.localStorage?
+{% endDetailsSummary %}
 
-You must declare the "storage" permission in the [extension manifest][doc-manifest] to use the storage
-API. For example:
+Even though extensions can access [Window.localStorage][mdn-localstorage] in some contexts (popup and other HTML pages), it is not recommended for the following reasons:
+
+- The extension's service worker cannot access the localStorage API.
+- Content scripts share the localStorage of the host page.
+- Data saved to localStorage is lost when the user clears their browsing history.
+
+{% endDetails %}
+
+### Storage areas
+
+The Storage API is divided into the following four buckets ("storage areas"): 
+
+- [**storage.local**][prop-local]
+  - Data is stored locally, which is cleared when the extension is removed.
+  - Quota limitation is approx 5 MB, but can be increased by requesting the `"unlimitedStorage"` permission.
+  - Consider using it to store larger amounts of data.
+
+- [**storage.sync**][prop-sync]
+  - If syncing is enabled, the data is synced to any Chrome browser that the user is logged into. If disabled, it behaves like `storage.local`.
+  - When the browser is offline, Chrome stores the data locally and resumes syncing when it's back online.
+  - Quota limitation is 100 KB approx, 8 KB per item.
+  - Consider using it to preserve user settings across synced browsers. 
+
+{% Aside 'warning' %}
+
+Local and sync storage areas should not store confidential user data because they are not encrypted at rest. When working with sensitive data, consider using the `session` storage area to hold values in memory until the browser is shut down.
+
+{% endAside %}
+
+- [**storage.session**][prop-session]
+  - Holds data in memory for the duration of a browser session.
+  - By default, not exposed to content scripts, but behavior can be changed by setting [chrome.storage.session.setAccessLevel()][method-access-level].
+  - Quota limitation is 1 MB approx.
+  - Consider using it to store global variables across service worker runs.
+
+- [**storage.managed**][prop-managed]
+  - Administrator can use a [schema][manifest-storage] and enterprise policies to configure a supporting extension's settings in a managed environment.
+  - Storage is read-only.
+
+## Manifest {: #manifest}
+
+To use the storage API, you must declare the `"storage"` permission in the extension
```

Comment:
```suggestion
To use the storage API, declare the `"storage"` permission in the extension
```

Reply:
'You must' is implied well enough by the context. This is an application of "omit needless words." (rule II-17)

Reply:
👍